internal/exec/resolvable/resolvable.go

```diff
@@ -380,13 +384,46 @@ func findMethod(t reflect.Type, name string) int {
 	return -1
 }
 
-func findField(t reflect.Type, name string) int {
+func findField(t reflect.Type, name string, index []int) []int {
 	for i := 0; i < t.NumField(); i++ {
-		if strings.EqualFold(stripUnderscore(name), stripUnderscore(t.Field(i).Name)) {
-			return i
+		field := t.Field(i)
+
+		if field.Type.Kind() == reflect.Struct && field.Anonymous {
+			newIndex := findField(field.Type, name, []int{i})
+			if len(newIndex) > 1 {
+				return append(index, newIndex...)
```

Comment:
I do not understand why we need to return an array.

Reply:
See https://github.com/graph-gophers/graphql-go/pull/338#discussion_r324255233 and https://github.com/graph-gophers/graphql-go/pull/338#discussion_r324257504.